libbeat/docs/newbeat.asciidoc

```diff
@@ -393,11 +412,16 @@ func main() {
 ----------------------------------------------------------------------
 
 [[ls-beat]]
-=== How to build Lsbeat from scratch
+=== Example: Building Lsbeat from Scratch
+
+https://github.com/kimjmin/lsbeat[Lsbeat] is similar to the `ls` command-line tool, but instead of printing the files and subdirectories to the screen,
+Lsbeat periodically ships them to Elasticsearch for storage. 
+
+To help you learn how to build a Beat, we've created this http://elastic.co//blog/build-your-own-beat[blog post]
+that describes how to build Lsbeat from scratch. The blog post includes additional details about fetching dependencies
+and compiling the code. 
 
-github.com/kimjmin/lsbeat[Lsbeat] is similar to the `ls` command line tool, but instead of printing the files and subdirectories to the screen,
-Lsbeat periodically ships them to Elasticsearch for storage.
-The http://elastic.co//blog/build-your-own-beat[blog post] describes how to build from scratch `Lsbeat`.
+//REVIEWERS: I wanted to add a little clarification here because it's initially jarring to come to this section in the doc because it talks about Lsbeat instead of the example that's used in the topics leading up to this one. I hope that flagging this section as an example makes the purpose less ambiguous.
```

Comment:
@dedemorton My idea was to include details about how to generate a Beat in the dev guide, and use the blog post for a concrete example (Lsbeat). That's why at the end of the dev guide, I am introducing Lsbeat to users to see how they can build a real example of a Beat. Feel free to change the content to make it more clear.